src/Infrastructure/LeanCode.ForceUpdate.Contracts/VersionSupport.cs

```diff
@@ -0,0 +1,29 @@
+using LeanCode.Contracts;
+
+namespace LeanCode.ForceUpdate.Contracts;
+
+public class VersionSupport : IQuery<VersionSupportDTO>
+{
+    public PlatformDTO Platform { get; set; }
+    public string Version { get; set; }
```

Comment:
If this and `Result` below were made nullable, we could allow querying for supported version range for callers that aren't actually those clients and don't have any version assigned.

But I'm not sure how much value is there in this use case (e.g. calling from Postman without faking the version?) so feel free to tell me there's no point and ignore this comment.

Reply:
> But I'm not sure how much value is there in this use case (e.g. calling from Postman without faking the version?) so feel free to tell me there's no point and ignore this comment.

That's not the use-case for this lib. It solves one single thing and we should stick with that.